docs/docsite/rst/scenario_guides/vmware_rest_scenarios/authentication.rst

```diff
@@ -0,0 +1,46 @@
+.. _vmware_rest_authentication:
+
+*******************************************
+How to configure the vmware_rest collection
+*******************************************
+
+.. contents:: Topics
+
+Introduction
+============
+
+The vcenter_rest modules need to be authenticated. There is two different
+ways to do this.
```

Comment:
```suggestion
```

Reply:
This seems like an odd edit, but it helps with translation to have complete sentences in one line.